_src/Anti-Debug/techniques/interactive.md

```diff
@@ -48,10 +49,10 @@ In the example below, we run the second instance of our process which tries to a
 
 bool IsDebugged()
 {
-    WCHAR wszFilePath[MAX_PATH], wszCmdLine[MAX_PATH];
-    STARTUPINFO si = { sizeof(si) };
-    PROCESS_INFORMATION pi;
-    HANDLE hDbgEvent;
+WCHAR wszFilePath[MAX_PATH], wszCmdLine[MAX_PATH];
```

Comment:
Please do not change the shifts in the code, revert all the similar cases to the spaces that were present initially